tools/osx-pkg-postinstall.sh

```diff
@@ -3,3 +3,4 @@
 # TODO Can we extract $PREFIX from the installer?
 cd /usr/local/bin
 ln -sf ../lib/node_modules/npm/bin/npm-cli.js npm
+ln -sf ../lib/node_modules/npm/bin/npx-cli.js npx
```

Comment:
IIRC this file doesn't actually run or something, but no harm in putting it here I suppose.